sdk_contrib/aws_v1_support/src/main/java/io/opentelemetry/sdk/contrib/trace/aws/Ec2Resource.java

```diff
@@ -16,16 +16,16 @@
 
 package io.opentelemetry.sdk.contrib.trace.aws;
 
-import com.amazonaws.util.EC2MetadataUtils;
-import com.amazonaws.util.EC2MetadataUtils.InstanceInfo;
 import io.opentelemetry.common.AttributeValue;
+import io.opentelemetry.sdk.contrib.trace.aws.resource.AwsResourcePopulator;
 import io.opentelemetry.sdk.resources.Resource;
-import io.opentelemetry.sdk.resources.ResourceConstants;
-import java.util.HashMap;
-import java.util.Map;
-import javax.annotation.Nullable;
 
-/** Provides for lookup and population of {@link Resource} labels when running on AWS EC2. */
+/**
+ * Provides for lookup and population of {@link Resource} labels when running on AWS EC2.
+ *
+ * @deprecated Use {@link AwsResourcePopulator}.
+ */
+@Deprecated
```

Comment:
since we're pre-1.0, I think we can just delete this if it's not the recommended way to do the work.